cmd/iso_package/config.go

```diff
@@ -0,0 +1,238 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/kdomanski/iso9660/util"
+)
+
+const (
+	tmpFolderPrefix = "iso_package_tmp"
+)
+
+type Config struct {
```

Comment:
I know naming is hard. But, why is this struct named `Config`? It's not only in charge of the configuration, it actually does the work

Reply:
Builder?